### DISCUSSION TITLE
Build error - CsWinRT requieres Universal Windows Platform

### DISCUSSION THREAD

**Discussion by chatay** [BornToBeRoot/NETworkManager#457] [Q&A] [answered]
running the .\build.ps1 and getting the below error. NET SDK 5.0  is already installed. any ideas?

EXEC : error : Could not find the Windows SDK in the registry [C:\Work\PersonelTraining\ping_1\Source\NETworkManager.Models\NETworkManager.Models.csproj]
C:\Users\cagat\.nuget\packages\microsoft.windows.cswinrt\1.1.1\build\Microsoft.Windows.CsWinRT.targets(111,5): error MSB3073: The command ""C:\Users\cagat\.nuget\packages\microsoft.windows.cswinrt\1.1.1\cswinrt.exe" @"C:\Work\PersonelTraining\ping_1\Source\NETworkManager.Models\obj\Release\net5.0-windows10.0.17763.0\win10-x64\Generated Files\cswinrt.rsp"" exited with code 1. [C:\Work\PersonelTraining\ping_1\Source\NETworkManager.Models\NETworkManager.Models.csproj]
C:\Work\PersonelTraining\ping_1\build.ps1 : Could not find dotnet release build. Is .NET SDK 5.0 or later installed?
At line:1 char:1
+ .\build.ps1
+ ~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,build.ps1

**Comment by BornToBeRoot** [comment, marked as answer]
You can check with `dotnet --version` in PowerShell if the latest version is used.

CsWinRT requires UWP `Universal Windows Platform development` to access modern APIs like `Windows.Devices.WiFi`. Maybe this is missing in your Visual Studio installation. 

https://github.com/microsoft/CsWinRT

**Reply by chatay** [reply]
**Universal Windows Platform** development was missing, once I installed it, it runs without any errors. Thanks!
